libnavigation-base/src/main/java/com/mapbox/navigation/base/speed/model/SpeedLimitUnit.kt

```diff
@@ -5,12 +5,17 @@ package com.mapbox.navigation.base.speed.model
  */
 enum class SpeedLimitUnit {
     /**
-     * Speed limit in kilometers per hour
+     * Posted speed limit in kilometers per hour
      */
     KILOMETRES_PER_HOUR,
 
     /**
-     * Speed limit in miles per hour
+     * Posted speed limit in miles per hour
      */
-    MILES_PER_HOUR
+    MILES_PER_HOUR,
+
+    /**
+     * Current speed in meters per second
+     */
+    METERS_PER_SECOND
```

Comment:
I believe adding new enum entries is considered a breaking change.

Reply:
Good call. Fixed/